src/ptvsd/adapter/server.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root
+# for license information.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from ptvsd.adapter import components
+
+
+class Server(components.Component):
+    """Handles the debug server side of a debug session."""
+
+    message_handler = components.Component.message_handler
+
+    class Capabilities(components.Capabilities):
+        PROPERTIES = {
+            "supportsConfigurationDoneRequest": False,
+            "supportsFunctionBreakpoints": False,
+            "supportsConditionalBreakpoints": False,
+            "supportsHitConditionalBreakpoints": False,
+            "supportsEvaluateForHovers": False,
+            "supportsStepBack": False,
+            "supportsSetVariable": False,
+            "supportsRestartFrame": False,
+            "supportsGotoTargetsRequest": False,
+            "supportsStepInTargetsRequest": False,
+            "supportsCompletionsRequest": False,
+            "supportsModulesRequest": False,
+            "supportsRestartRequest": False,
+            "supportsExceptionOptions": False,
+            "supportsValueFormattingOptions": False,
+            "supportsExceptionInfoRequest": False,
+            "supportTerminateDebuggee": False,
+            "supportsDelayedStackTraceLoading": False,
+            "supportsLoadedSourcesRequest": False,
+            "supportsLogPoints": False,
+            "supportsTerminateThreadsRequest": False,
+            "supportsSetExpression": False,
+            "supportsTerminateRequest": False,
+            "supportsDataBreakpoints": False,
+            "supportsReadMemoryRequest": False,
+            "supportsDisassembleRequest": False,
```

Comment:
nit: sorting
```
"supportsCompletionsRequest": False,
"supportsConditionalBreakpoints": False,
"supportsConfigurationDoneRequest": False,
"supportsDataBreakpoints": False,
"supportsDelayedStackTraceLoading": False,
"supportsDisassembleRequest": False,
"supportsEvaluateForHovers": False,
"supportsExceptionInfoRequest": False,
"supportsExceptionOptions": False,
"supportsFunctionBreakpoints": False,
"supportsGotoTargetsRequest": False,
"supportsHitConditionalBreakpoints": False,
"supportsLoadedSourcesRequest": False,
"supportsLogPoints": False,
"supportsModulesRequest": False,
"supportsReadMemoryRequest": False,
"supportsRestartFrame": False,
"supportsRestartRequest": False,
"supportsSetExpression": False,
"supportsSetVariable": False,
"supportsStepBack": False,
"supportsStepInTargetsRequest": False,
"supportsTerminateDebuggee": False,
"supportsTerminateRequest": False,
"supportsTerminateThreadsRequest": False,
"supportsValueFormattingOptions": False,


```

Reply:
The current order follows the order in the spec. But yeah, I don't think there's any particular reason to do that.